vars.pckrvars.hcl

```diff
@@ -0,0 +1,2 @@
+ami_prefix1 = "Blue_Ami"
```

Comment:
Is this file required as well since your variable already has a default value to use, in the `Blue_Green_Ami.pkr.hcl` file ?